Dockerfile

```diff
@@ -9,13 +9,10 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-# NOTE: using registry.access.redhat.com/rhel8/go-toolset does not work (user is requested to use registry.redhat.io)
-# NOTE: using registry.redhat.io/rhel8/go-toolset requires login, which complicates automation
-# NOTE: since updateBaseImages.sh does not support other registries than RHCC, update to RHEL8
-# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/devtools/go-toolset-rhel7
-FROM registry.access.redhat.com/devtools/go-toolset-rhel7:1.13.15-4  as builder
-ENV PATH=/opt/rh/go-toolset-1.13/root/usr/bin:${PATH} \
-    GOPATH=/go/
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
+FROM registry.access.redhat.com/ubi8-minimal:8.4-200 as builder
+RUN microdnf install -y golang unzip && \
```

Comment:
in the previous version of this PR, you installed 3 rpms: go, go-bin, and go-src. You only need the one here?

Reply:
ah, I see installing the `golang` RPM includes all three rpms, and enabled the go-toolset module too:
```
 golang-1.15.7-1.module+el8.4.0+9580+3b0e6c24.x86_64                                                                                 ubi-8-appstream                                  724.5 kB
 golang-bin-1.15.7-1.module+el8.4.0+9580+3b0e6c24.x86_64                                                                             ubi-8-appstream                                   94.1 MB
 golang-src-1.15.7-1.module+el8.4.0+9580+3b0e6c24.noarch   
```